Formula/linkerd.rb

```diff
@@ -0,0 +1,80 @@
+class Linkerd < Formula
+  desc "Drop-in RPC proxy designed for microservices"
+  homepage "http://linkerd.io/"
+  url "https://github.com/BuoyantIO/linkerd/releases/download/0.7.5/linkerd-0.7.5.tgz"
+  sha256 "6f18f77b6dac019e24ccfb6adec74d6b13430be79af05f504461b39db85ebdca"
+
+  depends_on :java => "1.8+"
+
+  def install
+    ENV.java_cache
+
+    # Replacing disco folder path
+    inreplace "config/linkerd.yaml", "disco", libexec/"disco"
+
+    # Installing and symlinking
+    libexec.install "disco", "docs", "linkerd-#{version}-exec"
```

Comment:
nit: Does it make sense to install the docs somewhere more "visible" like `pkgshare`?